src/pages/components/DisplayGapsPercentage.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+
+import { TEXTS } from '../../resources/texts'
+import './DisplayGapsPercentage.scss'
+
+function DisplayGapsPercentage(gapsPrecentage: number | undefined) {
+  if (!gapsPrecentage && gapsPrecentage != 0) return
+  let status = ''
+  switch (true) {
+    case gapsPrecentage === 0:
+      status = 'great'
+      break
+    case gapsPrecentage < 20:
+      status = 'average'
+      break
+    case gapsPrecentage >= 20:
+      status = 'terrible'
+      break
+  }
```

Comment:
are you sure about the numbers?
if the gap percentage is 1/10,000, will you consider it "average"?
also, let's try something cleaner
for example -
```js
const statuses = [
    {
        min: -Infinity,
        status: "invalid"
    },
    {
        min: 0,
        status: "great"
    },
    {
        min: 10,
        status: "average"
    },
    {
        min: 20,
        status: "terrible"
    },
    {
        min: 101,
        status: "invalid"
    },
]

function getStatus(percentage) {
    return statuses.findLast(status => status.min < percentage).status
}
```

Reply:
This also relates to my coloring method in the new "gaps patterns page", and I think we should be consistent with our coloring \ terms. So maybe, even something like that could be better and used by both pages:
```js
 const statuses = [
    {
        min: -Infinity,
        status: "invalid"
    },
    {
        min: 0,
        status: "great",
        color: "green" // or a more specific color code
    },
    {
        min: 10,
        status: "average",
        color: "orange"
    },
    {
        min: 20,
        status: "terrible",
        color: "red"
    },
    {
        min: 101,
        status: "invalid"
    },
]
```

Reply:
@ShayAdler I didn't understand to which page are you referring to because my changes are for the GapsPage as well.

The 'status' variable is for choosing one of these classes
```
&-great-result {
    color: $great-result;
  }
  &-average-result {
    color: $average-result;
  }
  &-terrible-result {
    color: $terrible-result;
  }
```
the variables (in scss file) contain
```
$great-result: #097d09;
$average-result: #b4b407;
$terrible-result: #b90a0a;
```
Did you mean to change the colors in the variables? as follows:
```
$great-result:  green;
$average-result: orange;
$terrible-result: red;
```

Reply:
@NoamGaash 
>if the gap percentage is 1/10,000, will you consider it "average"?

Yes, the green is for TEXTS.all_rides_completed 
the average is for missing some rides
and the red is for missing more than 20% of rides.
Make sense and this is also the requirements from the issue
>green - "כל הנסיעות בוצעו"
    yellow - < 20% miss
    red - >20%miss

It make sense because even if one bus will not make the ride as scheduled it can have some bad consequences on a group of people. They can late for work , for a meeting , take their child from school. So yeah, we count on the buses to take us to important places. When one bus will not make it on time we will need to wait another half an hour for the next bus (in some cities)

I prefer your cleaner code. Do you want me to make the boundaries configurable as well? I refer to the 10%,20% ?

Reply:
agreed, you're right